cirq/contrib/acquaintance/bipartite_test.py

```diff
@@ -0,0 +1,305 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import itertools
+
+import pytest
+
+import cirq
```

Comment:
Don't import cirq within cirq.

Reply:
I thought that was the convention within tests?

Reply:
Ah, my bad, I was mislead by the fact that there were imports within sub modules.

You should export all your methods up to cirq.contrib.acquaintances and then import that as ca or something.

Reply:
Done